src/util/prometheus.js

```diff
@@ -508,6 +508,18 @@ class Prometheus {
         type: 'gauge',
         labelNames: ['destType', 'feature'],
       },
+      {
+        name: 'braze_identify_calls_count_without_filter',
+        help: 'braze_identify_calls_count_without_filter',
+        type: 'gauge',
+        labelNames: ['destination_id'],
+      },
+      {
+        name: 'braze_identify_skipped_count',
+        help: 'braze_identify_skipped_count',
+        type: 'gauge',
```

Comment:
A gauge is a metric that represents a single numerical value that can arbitrarily go up and down.
Maybe a counter is what we need here